### ISSUE TITLE
Designing and Creating GitHub Issue Labels

### ISSUE THREAD

**Issue by halisbal** [bounswe/bounswe2023group5#7]
As we work together using GitHub issues, we need a clear way to organize and track our work. To achieve this, we have decided on a set of labels that we can use to better manage our issues in our meeting #1.

Here are the labels we have thought so far:

## Status Labels
* Assigned
* In Progress
* In Review
* Done
* Canceled

These labels are meant to indicate the current state of an issue. When an issue is created, it will initially be assigned the "Assigned" label. As work begins, the issue can be moved to "In Progress" and then "In Review" as it gets closer to completion. Once the issue is finished, it will be labeled as "Done". If the issue is no longer needed or is deemed unnecessary, it will be labeled as "Canceled".

## Type Labels
* Research
* Design
* Discussion
* Enhancement
* Bug
* Feature

These labels are meant to categorize the issue based on its content. For example, if an issue is related to a bug in our project, it will be labeled as "Bug". If it's related to a new feature we want to add, it will be labeled as "Feature". If it's a discussion about a new idea or a team meeting, it will be labeled as "Discussion".

## Priority Labels
* Low
* Medium
* High
* Critical

These labels are meant to indicate the level of importance of an issue. Issues labeled as "Critical" should be the top priority and require immediate attention. Issues labeled as "Low" can be addressed later when there is time.

I would like to invite our team members to share their ideas and feedback about these labels. Do you think there are any labels missing or anything that could be improved? Please let me know your thoughts.

**Comment by cisel-zumbul** [comment]
I think it may be a good idea to add *Status: New* tag for the issues that is not assigned to anyone yet. Also "Help Needed" tag would be useful to quickly take action if someone gets stuck on a task.

**Comment by halisbal** [comment]
> I think it may be a good idea to add _Status: New_ tag for the issues that is not assigned to anyone yet. Also "Help Needed" tag would be useful to quickly take action if someone gets stuck on a task.

Great suggestion, @cisel-zumbul 
Thank you for taking the time to provide your feedback and for suggesting these additional labels. I agree that a "Status: New" label will be helpful to identify issues that are not assigned to anyone yet, and a "Status: Help Needed" label will help team members quickly identify tasks where assistance is required. I have added both labels to our repository.

**Comment by halisbal** [comment]
> Another tag like Type: Documentation could also be added. Currently for wiki pages we are using Type: Design and that could cause some misunderstandings when software design issues are opened.

@UnalDeniz Thank you for your suggestion to add **Type: Documentation** as an issue label. I think it's a great idea, added it to our repository. This will help to avoid any confusion when opening software design issues.

**Comment by halisbal** [comment]
We have decided on the current issue labels as a team. That is why I am closing this issue.
